### ISSUE TITLE
Cannot import cytnx

### ISSUE THREAD

**Issue by manuschneider** [Cytnx-dev/Cytnx#307]
Hi,
I complied the current version of dev-master on my machine and got the following error:
NameError                                 Traceback (most recent call last)
Cell In[5], line 3
      1 import sys
      2 sys.path.insert(0,"/home/manuel/cytnx/Cytnx_lib")
----> 3 import cytnx

File ~/cytnx/Cytnx_lib/cytnx/__init__.py:4
      1 import os,sys
      2 from .cytnx import *
----> 4 if cytnx.__cytnx_backend__ == "torch":
      5     import numpy, torch
      7     from .Symmetry_conti import *

NameError: name 'cytnx' is not defined

This did not happen before and the conda packages work as well. Is it a problem with the new pytorch features? I switched pytorch off during compilation.

**Comment by jeffry1829** [comment]
I can compile & import cytnx without error.
May you provide the commit hash, the Install.sh, conda list,the python code you run?
Thanks!

**Comment by manuschneider** [comment]
I cannot reproduce the problem either somehow. Maybe something was wrong with my Install.sh script, I saw that new options were added in the new branch and I probably did not update my local script. Now it works fine.
Sorry for the trouble, but since this seems not reproducable I will close the issue.

Btw, the Install.sh in dev-master has different options enabled than stated in the user guide as default. I guess this is fine but should be changed for the next release. I suggest that we all use our local Install.sh for testing and do not change the one on github (once the correct defaults are restored.
